### PR TITLE
Update spanish books

### PR DESCRIPTION
**Pull request by miguelpalazzo**
## What does this PR do?
Add Resource(s)

### Description 
Introduce three (3) unexistent programming books to the Spanish list, each covering different topics of interest:
- NoSQL / Redis
- React
- Go

They're available as HTML (online version), and at least a PDF / EPUB version for downloading purposes. They were originally released in English, but made available as translated resources into Spanish, by Raúl Expósito and others.

### Why is this valuable (or not)?
Redis, React and Go are interesting topics to follow by any developer, and very active and on-demand since a few years ago in the community. However, not so many books get translated or even made available in Spanish language, making such knowledge hard to get for many on Spanish-speaking communities.

### How do we know it's really free?
They're released under Attribution-NonCommercial 3.0 Unported license.

### For book lists, is it a book? For course lists, is it a course? etc.
Books.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-es.md [hunk 1]
```diff
@@ -174,6 +176,11 @@
 * [Github Simple](https://github.com/MrOutis/GitHub-Simple#github-simple) (Github)
 
 
+### Go
+
+* [El pequeño libro Go](https://raulexposito.com/the-little-go-book-en-castellano.html) (HTML, PDF, EPUB)
```
Comment (eshellman) [inline free-programming-books-es.md:181]: Please add author names where appropriate.

File: free-programming-books-es.md [hunk 2]
```diff
@@ -241,6 +248,7 @@
 #### React
 
 * [Desarrollo de Aplicaciones Web con React.js y Redux.js](https://leanpub.com/react-redux) (PDF, EPUB, MOBI)
+* [SurviveJS - React de aprendiz a maestro](https://es.survivejs.com) (HTML, PDF)
```
Comment (eshellman) [inline free-programming-books-es.md:251]: Please add author names where appropriate.

File: free-programming-books-es.md [hunk 3]
```diff
@@ -270,6 +278,11 @@
 * [MongoDB en español: T1, El principio](https://dpdc.gitbooks.io/mongodb-en-espanol-tomo-1/content) - Yohan Graterol (Gitbook) (:construction: *en proceso*)
 
 
+#### Redis
+
+* [El pequeño libro Redis](https://raulexposito.com/the-little-redis-book-en-castellano.html) (HTML, PDF, EPUB)
```
Comment (eshellman) [inline free-programming-books-es.md:283]: Please add author names where appropriate.

**Comment by miguelpalazzo** [comment]
I have added the author names, while also updated from the upstream.

Please, let me know if there is anything else to be done! Thanks in advance @eshellman

**Comment by eshellman** [comment]
Beautiful!
Thanks, and welcome to free-programming-books!

**Pull request merged**